Upstream: Should be
Reason: Fixes version in pkg-config files

diff --git a/meson.build b/meson.build
index 9e89384..e9fa6e1 100644
--- a/meson.build
+++ b/meson.build
@@ -1,5 +1,5 @@
 project('rofi', 'c',
-    version: '1.5.4-dev',
+    version: '1.6.0',
     meson_version: '>=0.47.0',
     license: [ 'MIT' ],
     default_options: [
